Documentation/kube-flannel.yml

```diff
@@ -305,9 +327,20 @@ spec:
         tier: node
         app: flannel
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: beta.kubernetes.io/os
```

Comment:
Need to be 1.16 compliant. @arasananbu This api extension changed, isn't it?

Reply:
Can take care of this in another PR. Nevermind.

Reply:
Yes, I have reviewed the official doc of kubernetes. nodeAffinity needs to be supported by default in a higher version. Do I need to re-deliver a pr and change nodeAffinity to nodeSelector?

Reply:
Learned by kubernetes release note, Nodes and pods support affinity and anti-affinity in kubernetes 1.6[releasetnotes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.6.md#major-updates-and-release-themes)

Do I still need to adjust? change nodeAffinity to nodeSelect? @rajatchopra @arasananbu